src/components/Calculator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './Calculator.css';
+
+export default function Calculator() {
+  return (
+    <>
```

Comment:
- [ ] Your `Calculator.js` component should be class-based, Please update that 

![image](https://user-images.githubusercontent.com/75126481/153275258-2ae2676d-4b62-4a86-94b1-107aa895c026.png)